docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3334,7 +3339,9 @@ async function toggleBrb() {
   const meeting = getCurrentMeeting();
 
   if (meeting) {
-    const enabled = document.getElementById('brb').checked;
+    const brbButton = document.getElementById('brb-btn');
+    const enabled = brbButton.innerText !== 'Remove away';
```

Comment:
In the previous line we have `brbButton` so I would suggest renaming this to `brbEnabled` so that `enabled` doesn't sound ambiguous.

Reply:
Thanks for idea, I feel good with that